File: src/main/java/eu/happybe/openapi/mysql/query/DestroyPartyQuery.java
package eu.happybe.openapi.mysql.query;

import eu.happybe.openapi.mysql.AsyncQuery;

import java.sql.SQLException;
import java.sql.Statement;

public class DestroyPartyQuery extends AsyncQuery {

    public String owner;

    public DestroyPartyQuery(String owner) {
        this.owner = owner;
    }

    @Override
    public void query(Statement statement) throws SQLException {
        statement.executeUpdate("DELETE FROM HB_Parties WHERE Owner='"+this.owner+"';");
    }
}

File: src/main/java/eu/happybe/openapi/mysql/query/DestroyPartyQuery.kt
package eu.happybe.openapi.mysql.query

import eu.happybe.openapi.mysql.AsyncQuery
import java.sql.SQLException
import java.sql.Statement

class DestroyPartyQuery(var owner: String) : AsyncQuery() {
    @Throws(SQLException::class)
    override fun query(statement: Statement) {
        statement.executeUpdate("DELETE FROM HB_Parties WHERE Owner='" + owner + "';")
    }
}